src/serialization/pkl2proto.py

```diff
@@ -0,0 +1,155 @@
+import pickle
+import sys
+import numpy as np
+import font_pb2
+import serialization 
+
+def parseDict(d, filename= "No filename provided"):
+    '''
+    Read a Dict containing font curves and return the info in a standardized np array
+
+    Parameters
+    ----------
+    d: Python Dictionary with key of the chars and the value of a closed curves dict, 
+    the closed curve dict has keys of ints up to max num closed curves and lists
+    of assosiated bez curves. Each bez curve in the list of bez curves has 3 coordinates
+    which are x and y coords of the control points. 
+    
+    filename: Optional parameter for knowing where the reading of the data failed
+    
+    
+    Returns
+    -------
+    New NP array of size len(chars), maxClosedCurves, maxCurvePerGlyph, 3, 2
+    '''
+
+     chars = [
```

Comment:
This is actually my bad here, but I forgot 'W' I will add it for parsettx.py and send up a pull request, but can you add it here too? 
@eigenfoo  Can you sanity check and make sure that is the only other letter we are missing here?

Reply:
@Jped just approved your PR.

@ostapstephan yeah you should also add the W. This will probably impact our accuracy scores.